conf_files/pocs.yaml

```diff
@@ -91,14 +91,14 @@ observations:
 # By default all images are stored on googlecloud servers and we also
 # use a few google services to store metadata, communicate with servers, etc.
 #
-# See $PANDIR/panoptes/utils/google/README.md for details about authentication.
+# See $POCS/pocs/utils/google/README.md for details about authentication.
 #
 # Options to change:
 #   image_storage: If images should be uploaded to Google Cloud Storage.
 #   service_account_key: Location of the JSON service account key.
 ################################################################################
 panoptes_network:
-    image_storage: False
+    image_storage: True
```

Comment:
We don't want to default this to `True` for everyone. We will have people change it to `True` once they have a valid service account key and so they actually have the ability to upload.